src/test/scala/tech/cryptonomic/conseil/tezos/TezosDatabaseOperationsTestFixtures.scala

```diff
@@ -721,9 +722,14 @@ trait TezosDataGeneration extends RandomGenerationKit {
         )
       )
 
+    val sampleBallot =
+      Ballot(
+        ballot = Vote("yay")
+      )
+
     val sampleOperations =
       sampleEndorsement :: sampleNonceRevelation :: sampleAccountActivation :: sampleReveal :: sampleTransaction :: sampleOrigination :: sampleDelegation ::
-          DoubleEndorsementEvidence :: DoubleBakingEvidence :: Proposals :: Ballot :: Nil
+          DoubleEndorsementEvidence :: DoubleBakingEvidence :: Proposals :: sampleBallot :: Nil
```

Comment:
Same observation here..

Reply:
It's because of the change from case class to case object.